R/pkg/R/functions.R

```diff
@@ -1339,7 +1339,7 @@ setMethod("pmod", signature(y = "Column"),
 #' @export
 setMethod("approxCountDistinct",
           signature(x = "Column"),
-          function(x, rsd = 0.95) {
+          function(x, rsd = 0.05) {
```

Comment:
Yeah looks like it should be 0.05 -- cc @davies 
This seems to have been added back when SparkR was a separate code-base in https://github.com/davies/SparkR-pkg/commit/d7b17a428c27aac28d89e1c85f1ba7d9d4b021d2